java/src/Ice/src/main/java/com/zeroc/Ice/DatagramLimitException.java

```diff
@@ -20,21 +7,13 @@
  * configured send or receive buffer size, or exceeds the maximum payload size of a UDP packet
  * (65507 bytes).
  */
-public class DatagramLimitException extends ProtocolException {
+public final class DatagramLimitException extends ProtocolException {
   public DatagramLimitException() {
-    super();
-  }
-
-  public DatagramLimitException(Throwable cause) {
-    super(cause);
+    this("");
```

Comment:
> This doesn't look correct.

I fixed the message

> The C# version has a single ctor

This is because of a difference in the UdpTransceiver.
In C#, we just use the default message:
`throw new Ice.DatagramLimitException();`
but in Java, we also include the packetSize in the message:
`throw new com.zeroc.Ice.DatagramLimitException("message size of " + buf.size() + " exceeds the maximum packet size of " + packetSize);`

In C++, we don't include this extra information:
`throw DatagramLimitException(__FILE__, __LINE__);`
So, Java is the odd-one-out here.
But this information seems useful to have... If you exceeded a limit, it's good to know what the limit was, right?